module_1/src/com/epam/cycles/CyclesTaskA.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.cycles;
+import java.util.Scanner;
+
+public class CyclesTaskA {
+
+    public static void main(String[] args) {
+
+    Scanner console = new Scanner(System.in);
+    System.out.print ("Enter the natural number A (non-negative integer greater than or equal to 1): ");
+    int a = console.nextInt();
+
+        if (a >= 1) {
+            int sum = 0;
```

Comment:
let's change the type of variable from "int" to "long" to make the possibility to count bigger numbers
```suggestion
    long a = console.nextLong();

        if(a >= 1) {
            long sum = 0;
```